gdal_subsetter/transform.py

```diff
@@ -695,39 +706,9 @@ def reformat(self, transformed_geotiff_file: str, dstdir: str) -> str:
 
         return formatted_file_path
 
-    def get_variables(self, filename: str):
-        """filename is either nc or tif."""
-        gdalinfo_lines = gdal.Info(filename).splitlines()
-
-        result = []
-        if "netCDF" in gdalinfo_lines[0] or "HDF" in gdalinfo_lines[0]:
-            # netCDF/Network Common Data Format, HDF5/Hierarchical Data Format
-            # Release 5
-            # Normal case of NetCDF / HDF, where variables are subdatasets
-            for subdataset in filter(
-                (lambda line: re.match(r"^\s*SUBDATASET_\d+_NAME=", line)),
-                gdalinfo_lines,
-            ):
-                result.append(HarmonyVariable({"name": re.split(r":", subdataset)[-1]}))
```

Comment:
This is what started me off with this PR - I spotted code that was trying to accommodate netCDF4 input files. But those are no longer handled, so this code is unreachable.